docs/dv/02_validator-creation.md

```diff
@@ -10,8 +10,23 @@ description: Creating a Distributed Validator cluster from scratch
 
 To create a distributed validator cluster, you and your group of operators need to complete the following steps:
 
-1. Each operator must set up their staking software on their node, expose the necessary ports to the internet, and configure appropriate monitoring, logging and alerting.
-2. One operator leads, and all operators participate in a [Distributed Validator Key](../dvk/01_distributed-validator-keys.md) generation ceremony.
-3. All operators load the keys and manifests generated in the ceremony, into their staking deployments.
-4. Operators can run a performance test of the configured cluster to ensure connectivity between all operators at a reasonable latency is observed.
-5. Once all readiness tests have passed, one person activates the distributed validator(s) with a deposit on chain.
+1. One operator begins the cluster setup on the [Distributed Validator Launchpad](../dvk/02_distributed_validator_launchpad.md)
+    - This involves setting all of the terms for the cluster, including; withdrawal address, fee recipient, validator count, operator addresses, etc. This information is known as a `cluster configuration`.
+    - This operator also sets their charon client's Ethereum Node Record (ENR).
+    - This operator signs both the hash of the cluster config and the ENR to prove custody of their address.
+    - This data is stored in the DV Launchpad data layer and a URL is generated. This is a link for the other operators to join and complete the ceremomy.
```

Comment:
what does `data layer` mean?

Reply:
Me not wanting to promise IPFS support so soon but not wanting to lock in centralised DB either 😅

Reply:
Optimum choice!